.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
This is also a suggestion: (not merge blocking):

It might be a good idea to automatically add labels as well, e.g. see https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

However, I think that can be omitted for now and needs some more discussion about the labels in general.

However, if we use the same templates in all docs repos, we need same labels there (if we want to reuse the issue templates with labels). That should probably be the case for the defaults,  but I am not sure.

For now, it might make sense to just go with the version without labels and adjust that later.  (I added the labels "content missing", "content outdated" etc. to the repos a while ago but the question what is really helpful and I think this needs more discussion.

Also, the difference between bug and feature is also not so clear to me for docs. What is if content is missing. Is that a bug or a feature? Wrong content or content outdated is IMHO more clear - is bug.

https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Editors/issues/labels

Reply:
I don't think we need labels as long we don't get many issues from outside, so we have to mark them explicitly and separate them from issues from "inside".